dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

Comment:
Need to check if Dependabot scans leverage our workflow minutes, if so, probably want to bump this to monthly.

Reply:
https://github.blog/news-insights/product-news/dependabot-on-github-actions-and-self-hosted-runners-is-now-generally-available/

According to this blog post, dependabot will not contribute towards github action minutes.